test/local/devices.js

```diff
@@ -316,9 +316,9 @@ describe('lib/devices:', () => {
             assert.equal(log.activityEvent.args[0][0].is_placeholder, true, 'is_placeholder was correct')
 
             assert.equal(log.info.callCount, 1, 'log.info was called once')
-            assert.equal(log.info.args[0].length, 1, 'log.info was passed one argument')
-            assert.deepEqual(log.info.args[0][0], {
-              op: 'device:createPlaceholder',
+            assert.equal(log.info.args[0].length, 2, 'log.info was passed two arguments')
```

Comment:
Just for future reference, I've been gradually deleting these final description arguments to assertions as I encounter them.

They're a hangover from pre-chai days when the default assertion messages were unhelpful, now they just add a maintenance burden. Not saying do anything now, just something to bear in mind in the future.